tests/HttpExceptionTests.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace Tests\Tnapf\Router;
+
+use PHPUnit\Framework\TestCase;
+use Psr\Http\Message\ServerRequestInterface;
+use Tnapf\Router\Exceptions\HttpInternalServerError;
+use Tnapf\Router\Exceptions\HttpException;
+use RuntimeException;
+use HttpSoft\Message\ServerRequest;
+
+class HttpExceptionTests extends TestCase {
+    public ServerRequestInterface $request;
+
+    public function getMissingDescriptionHttpException(): HttpException
+    {
+        $this->request ??= new ServerRequest([], [], [], [], [], "GET", "/");
+
+        return new class($this->request) extends HttpException
```

Comment:
~Just me commenting yet again that this shouldnt use properties but instead an interface with functions like `getCode()` and `getPhrase()` which would remove the need for this test entirely, but I don't think I'm gonna change your mind this time around~

Reply:
Http exceptions will probably be removed from this lib in future versions which is why I'm not too worried about it atm